### DISCUSSION TITLE
Add the possibility to host the AIO admin interface behind a reverse proxy (Nginx) but in a subfolder/subdirectory

### DISCUSSION THREAD

**Discussion by daks** [nextcloud/all-in-one#5480] [Ideas] [answered]
Comes from the discussion here https://help.nextcloud.com/t/nextcloud-aio-interface-behind-nginx-in-a-subfolder/207327

I tried to host the AIO admin interface in a subfolder like  https://domain.example.com/subfolder (I plan to soon have two AIO instances and maybe more, and hosting them all in the same domain would be great) but didn't find how, I think it needs code changes to make it possible, like the `OVERWRITEWEBROOT` parameter in the nextcloud server code.

Is there any possibility to see it implemented? 
_(I would help if I had the PHP skills..)_,

**Comment by szaimen** [comment, marked as answer]
Hi, are these instance running in a home network? If so, I would suggest not exposing the AIO interface publicly and rather putting it behind a VPN for better security.

**Reply by daks** [reply]
This is a solution in fact.
But today it's already possible to expose it on the internet, just not in a subfolder, so I don't really see any difference :)

I still think it could be an interesting possibility, either in private networks or when you can't use a VPN (in restrained networks, public wifi...).

If it's because the AIO admin interface is considered unsecure (or particularly vulnerable), maybe a warning should be added to the webpage.

**Reply by szaimen** [reply]
The aio interface is not considered insecure but it is still safer to not expose it to the public internet if possible.

**Reply by daks** [reply]
That's why I want to host it under my 'admin' domain, which is protected by authentication :)

**Reply by szaimen** [reply]
I see so the instances are not running in a home network?

**Reply by daks** [reply]
The Nextcloud server you mean? No

**Reply by szaimen** [reply]
I see

**Reply by daks** [reply]
@szaimen do you think there is any chance to see this feature implemented?

**Reply by szaimen** [reply]
I fear we dont have any plans to implement this ourselves. However PRs are welcome. I've also just added this to https://github.com/nextcloud/all-in-one/issues/5251

**Reply by daks** [reply]
Then I may try to find how it can be done if I'm able to.
Thanks for your answer.

**Comment by daks** [comment]
link to https://github.com/nextcloud/all-in-one/discussions/839
